src/elements/content-uploader/README.md

```diff
@@ -26,6 +26,7 @@ var ContentUploader = require('./ContentUploader').default;
 | onBeforeUpload | function(Array&lt;[File](https://developer.box.com/reference#file-object)&gt;) |  | Callback function for retrieving an item before it has uploaded on files only, doesn't work on folders|
 | onClose | function |  | Callback function for when the close button is pressed. |
 | onComplete | function(Array&lt;[File](https://developer.box.com/reference#file-object)&gt;) |  | Callback function for when uploads are complete. |
+| onProgress | function([File](https://developer.box.com/reference#file-object)) |  | Callback function for handleUploadProgress. |
```

Comment:
Ideally, these instructions wouldn't be tied to a particular consuming application's implementation. Can this be a little more generic, similar to the descriptions above and below it?

Reply:
Updated!